react/components/Autocomplete/index.tsx

```diff
@@ -135,8 +135,17 @@ class AutoComplete extends React.Component<
     );
   }
 
+  addTermToHistory() {
+    const path = window.location.href.split("_q=");
```

Comment:
I recommend using the `useSearchPage` hook to get the search term. It Is a context with all the info related to the search

Reply:
I tried to use this, but it is always returning `undefined` for the Autocomplete component. I think this only works for children of the search result :/